planetary-social/nos
# Replace remaining rgb Color references with semantic Color assets

## Issues covered
https://github.com/planetary-social/nos/issues/1375 (ongoing work)

## Description
Replaced remaining rgb Color references with semantically named colors in Colors.xcassets

## How to test
- Verify that the rgb colors in each asset match the hard coded rgb colors in the changed files; smoke test the app to verify
